goltsmannn2/flake.nix

```diff
@@ -0,0 +1,23 @@
+{
+  description = "a set of macros to make cmake easier";
+
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs";
+  };
+
+  outputs = { self, nixpkgs, ... } @inputs:
+      let
+            system = "aarch64-darwin";
+            pkgs = import nixpkgs {
+                system = system;
+            };
+            hello_world_pkg = pkgs.callPackage ./default.nix {};
+      in
+      {
+        packages.aarch64-darwin = rec {
+            default = hello_world_pkg;
+        };
```

Comment:
this is a good flake! except for one thing which is we should prefer to support multiple system types per flake:

you can do this following this blog post's outline
https://ayats.org/blog/no-flake-utils